src/App.jsx

```diff
@@ -0,0 +1,30 @@
+import React, {useState} from "react";
+import { BrowserRouter as Router, Redirect, Switch, Route } from "react-router-dom"
+import { SpotifyLogin } from "./SpotifyLogin";
+import { SpotifyAuthRedirect } from "./SpotifyAuthRedirect";
+import { SpotifyPlaylists } from "./SpotifyPlaylists";
+
+export const App = () => {
+    const [loggedIn, setLoggedIn] = useState(false);
```

Comment:
According Trello you should also show logged in username. So you need to keep user token in a store to be able to reuse it in different components (to making Spotify API calls).